pkg/roachprod/prometheus/prometheus.go

```diff
@@ -436,7 +436,7 @@ func Shutdown(
 		"stop prometheus",
 		"sudo systemctl stop prometheus || echo 'Stopped prometheus'",
 	); err != nil {
-		l.Printf("Failed to stop prometheus server")
+		l.Printf("Failed to stop prometheus server: %v", err)
```

Comment:
Return err to the caller.

Reply:
Ah I see. I used `errors.CombineErrors` to ensure the function would try shutting down everything it possibly could. Note that tpcc roachtests no longer throw a fatal error after prom/grafana shutdown, now that these shutdown errors get returned to it. I can change this back after this PR ships, but I don't want to cause a flurry of roachtest failures due to grafana shutdown.

Reply:
irfan approved offline.